plugin.php

```diff
@@ -76,16 +76,40 @@ function register_json_route( $namespace, $route, $args = array(), $override = f
 }
 
 /**
+ * Add the extra Post Type registration arguments we need
+ * These attributes will eventually be committed to core.
+ */
+function _add_extra_api_post_type_arguments() {
+	global $wp_post_types;
+
+	$wp_post_types['post']->access_in_json = true;
+	$wp_post_types['post']->json_base = 'posts';
+	$wp_post_types['post']->json_controller_class = 'WP_JSON_Posts_Controller';
+
+	$wp_post_types['page']->access_in_json = true;
+	$wp_post_types['page']->json_base = 'pages';
+	$wp_post_types['page']->json_controller_class = 'WP_JSON_Posts_Controller';
+
+	// $wp_post_types['attachment']->access_in_json = true;
+	// $wp_post_types['attachment']->json_base = 'media';
+	// $wp_post_types['attachment']->json_controller_class = 'WP_JSON_Attachments_Controller';
+
+}
+add_action( 'init', '_add_extra_api_post_type_arguments', 11 );
```

Comment:
Could be, why?

I added them just to get things working, with the assumption they'd eventually be committed to `create_initial_post_types()`